src/buildercore/cfngen.py

```diff
@@ -418,14 +419,23 @@ def non_empty(self):
 _empty_cloudformation_dictionary = {'Resources': {}, 'Outputs': {}}
 Delta.__new__.__defaults__ = (_empty_cloudformation_dictionary, _empty_cloudformation_dictionary, _empty_cloudformation_dictionary, None)
 
+class TerraformDelta(namedtuple('TerraformDelta', ['old_contents', 'new_contents'])):
+    def __str__(self):
+        return self.new_contents
+
+    def diff(self):
+        return deepdiff.DeepDiff(json.loads(self.old_contents), json.loads(self.new_contents))
+
 def template_delta(context):
     """given an already existing template, regenerates it and produces a delta containing only the new resources.
 
     Some the existing resources are treated as immutable and not put in the delta. Most that support non-destructive updates like CloudFront are instead included"""
     old_template = read_template(context['stackname'])
     template = json.loads(cloudformation.render_template(context))
+    old_terraform_template_file = '{}'
```

Comment:
weird magic literal, it's beginning to smell a lot like eval. or php. A simple `empty_template = '{}'` and then `old_terraform_template_file = empty_template` would do